script/1_0_2current_en_script/Arcueid/Day 7/07_00_ARC07_1.txt

```diff
@@ -29,74 +29,75 @@ Perhaps it's just me, but her bow doesn't seem as sharp as usual.
+The beginning of the week. A good chance for a fresh start.
 <Page14>
 I wash my face and fix my clothes before entering the living room.
 Akiha is sitting on the sofa.
-It is just past 7 AM, but it seems that she already finished breakfast and is now gracefully enjoying a cup of tea.
+It's just past 7 AM, but it seems like she has already finished breakfast and is now gracefully enjoying a cup of tea.
 <Page15>
 "Oh, good morning, Nii-san. You are up early this morning."
-Whether it was because she was pleased to see me up so early, Akiha seems to be in a good mood.
+Possibly because she was pleased to see me up so early, Akiha seems to be in a good mood.
 "Good morning. Likewise, you seem to be up early as alwa―――"
 <Page16>
-...And then. #I suddenly feel lightheaded and push down on the corner of my eyes.
-And even then, the dizziness won't go away.
-Without knowing what's going on, my body temperature rises and―――
+...And then. 
+I suddenly feel lightheaded, so I push down on the corner of my eyes.
+But even then, the dizziness won't go away.
+My body temperature rises up to the point that even I think something is wrong and―――//　自分でもどうかと思うぐらい、体温が、上がっていく―――
 <Page17>
-"Nii-san...?"
-I hear the faint rustle of clothes and the sound of approaching footsteps.
+"Nii-san...?"//　
+I hear the faint rustling of clothes and the sound of approaching footsteps.
 <Page18>
 "Is something wrong? Your face is red, do you have a fever?"
 "Ah... no, it's true that I have a fever, but..."
 <Page19>
 Akiha approaches me without reservation and peers at my face from below.
 ...That's odd.
-For some reason, when I look at Akiha, it feels like my dizziness is getting stronger.
+For some reason, when I look at Akiha, it feels like my dizziness gets even more intense.
 <Page20>
 "...It seems that you have a bit of a fever.
-Did you make sure to use your blanket last night? I am sure it is just your condition being deteriorated because of the change in environment, but just to be sure.
-Please wait here, Nii-san. I'm calling Kohaku right now, so we will measure your temperature before having breakfast."
+Did you make sure to use your blanket last night? I am sure it is just your condition deteriorating because of the change in environment, but just to be sure.
+Please wait here, Nii-san. I will call Kohaku right now, so let us measure your temperature before having breakfast."
 <Page21>
 Akiha turns her gaze to the lobby. She must be going to call Kohaku-san, who is preparing breakfast in the kitchen.
-I take this opportunity to look away from Akiha.
-...As I thought, my dizziness is alleviated with that action alone.
+I take this opportunity to avert my gaze from Akiha.
+...As I thought, my dizziness was alleviated with that.
 <Page22>
 "It's okay, Akiha. You don't need to call Kohaku-san.
-I wasn't feeling well because I just woke up, so you don't have to get so worried―――"
+I wasn't feeling well because I just woke up, so you don't have to be so worried―――"
```

Comment:
'just' repetition notwithstanding, I think another word needs to go before "wasn't otherwise it sounds like something else is going to follow:
>  I just wasn't feeling well because I just woke up, you don't need to be so worried.